### PR TITLE
Create what-is-open-liberty.adoc

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: ref/general/what-is-open-liberty.adoc [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+:seo-description: Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+:page-layout: general-reference
+:page-type: general
+= What is Open Liberty
+
+Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+
+With Open Liberty, it's easy to add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile. This modular structure simplifies microservice development, so you run Just enough Application Server to support the features that your application needs. Furthermore, with Open Liberty zero migration architecture, you can upgrade to the latest version with minimal impact to your current applications and configurations. Open Liberty is compatible with the Jakarta EE 8 Full Platform and Web Profile specifications and with MicroProfile 3.0. Open Liberty is deployed on every major developer platform, including Docker, Kubernetes, and Cloud Foundry. For the latest updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
```
Comment (ManasiGandhi) [inline ref/general/what-is-open-liberty.adoc]: Change,
"With Open Liberty, it's easy to add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile."
to,
"With Open Liberty, you can add and remove modular features from the latest versions of Jakarta EE and Eclipse MicroProfile."
Acrolinx is flagging the word 'easy'. Also, the word 'latest' which appears thrice, If there is any way to replace it, though the third occurence "For the latest updates" should be okay I guess.

Change,
"so you run Just enough Application Server to support the features that your application needs."
to,

"so you run only the required Application Server to support the features that your application needs."

Acrolinx is flagging 'just enough'

Also,
If possible replace the word 'platform' in Open Liberty is deployed on every major developer platform. Acrolinx flags it.

Comment (dmuelle) [reply ref/general/what-is-open-liberty.adoc]: Thanks Manasi- I removed "easy" but have further comments for the others-

"Just enough Application Server" is a microservice buzz-term Alasdair requested as part of this description. Leaving it in for now

I think "latest" is ok if not referencing a specific feature or product. It's used in a general sense here.

"Platform" acrolinx recommends using this to mean "operating system". The use here is different. Not though if there is a better word for Docker, etc- which identify as Platform-as-a-service (PlaaS)

Added some more info based on sources in the issue

File: ref/general/what-is-open-liberty.adoc [hunk 2]
```diff
@@ -16,4 +16,24 @@
...
+* A lightweight edition that corresponds to the Java EE Web Profile specification.
+* Excellent development and production runtime environments for web applications.
+* A smaller footprint for faster download and startup, giving more development time and faster time to deployment.
+* Ease of packaging applications for deployment, including configuration.
+* The ability to run applications that are written for Open Liberty on WebSphere Application Server traditional.
+* The ability to extend the Open Liberty capabilities by adding custom features that use a product extension System Programming Interface (SPI).
+
+== Server configuration
+
+Open Liberty follows a principle of configuration by exception, which allows for succinct config files. The runtime environment operates from a set of built-in config default settings. You only specify config that overrides the default settings. For more information, see link:/docs/ref/config/#serverConfiguration.html[Server Configuration].
```
Comment (ManasiGandhi) [inline ref/general/what-is-open-liberty.adoc]: Change,
"You only specify config that overrides the default settings. "
to,
"You specify only the config that overrides the default settings. "

File: ref/general/what-is-open-liberty.adoc [hunk 3]
```diff
@@ -16,4 +16,24 @@
...
+
+== Feature Management
+
+Features are the discrete units of functionality by which you control the pieces of the runtime environment that are loaded into a particular server. Features provide the programming models and services that support your applications. For a full list of avaialble features and their configuration elements, see link:/docs/ref/feature/#featureOverview.html[Server Features].
```
Comment (ManasiGandhi) [inline ref/general/what-is-open-liberty.adoc]: Change
"Features are the discrete units of functionality by which you control the pieces of the runtime environment that are loaded into a particular server. "
to
"Features are the discrete units of functionality with which you control pieces of the runtime environment that are loaded into a particular server. "

Acrolinx flagged 'functionality'. Use your discretion for appropriate use per technical accuracy.

Also, a minor typo on the word 'avaialble'

File: ref/general/what-is-open-liberty.adoc [hunk 4]
```diff
@@ -16,4 +16,24 @@
...
+
+== Open Liberty Guides
+
+Open Liberty provides a diverse library of interactive and static user guides that are designed to give you firsthand experience with the configuration and features that support cloud native development. The guides cover a wide range of development and production use cases and can be adapted as reference templates for developing your own applications. link:/guides/[Browse our guides] and learn what Open Liberty can do for you.
```
Comment (ManasiGandhi) [inline ref/general/what-is-open-liberty.adoc]: Acrolinx flags 'our' in [Browse our guides].

**Comment by dmuelle** [comment]
Changes addressed

**Pull request merged**